### PR TITLE
screen capture updates and mac version change

### PR DESCRIPTION
**Pull request by moxiegirl**
Updating with the new screen shots and changes from Jeff.

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @thaJeztah

File: docs/mac/step_one.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ attach to a Docker VM on your machine. This VM hosts Docker for you on your Mac.
 
 ## Step 1: Check your version
 
-Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Docker.
+Your Mac must be running OS X 10.8 "Mountain Lion" or newer to run Docker.
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: s/to run Docker/to run Docker Toolbox/

(Don't know how precise you want to be, but `docker` (client) does run on older versions, it's just that VirtualBox doesn't)

**Comment by thaJeztah** [comment]
Thanks @moxiegirl one minor nit, also (if you want); I optimized the images for you, saving roughly 40% in size. You can download them from WeTransfer: https://we.tl/iAN0j8x9TM

LGTM (with or without my comment addressed)

**Comment by moxiegirl** [comment]
@thaJeztah Thanks. This is a merge into `docs` for deploy.  We'd need to make changes in the master.

**Comment by thaJeztah** [comment]
Doh! sorry, missed that :blush:

LGTM, but you'll have to merge, cause I don't have permissions here

**Comment by moxiegirl** [comment]
LGTM @thaJeztah **gotta** get that fixed.

**Pull request merged**